### PR TITLE
Use single quotes instead of double quotes

### PR DESCRIPTION
**Pull request by hwrdtm**
# Why

tl;DR - Double quotes are not parseable by siwe-parser per the ABNF grammar specified in [siwe-parser](https://github.com/spruceid/siwe/tree/main/packages/siwe-parser).

Currently we are generating double quote marks `"` which is not part of the grammar that is specified in the siwe-parser library (https://github.com/spruceid/siwe/blob/main/packages/siwe-parser/lib/abnf.ts#L108-L112). The result is that, when I try to pass a SIWE message (with caps declared in the statement) as a string into `new SiweMessage()`, the parser is used and it throws an error saying the SIWE message doesn't conform to the ABNF / specified grammar. I've confirmed this with a new test case with an added `"` in the parsing_positive.json in your siwe package, as an MCVE.

The reason why the tests pass in recap-ts right now is because we're passing in an object into `new SiweMessage` so the parser is not used, and hence tests pass. Perhaps we should add some tests in recap-ts where the SIWE message is in the form of a string too.

# What

Use single quote marks instead.

Add test case for passing string to `new SiweMessage()` constructor.

# Testing

`yarn test`.

### CODE REVIEW

**Review by chunningham — APPROVED** [review]
nice, thanks for the contrib, we will merge and release this asap
